test/features/gen_target.py

```diff
@@ -0,0 +1,76 @@
+import rasterio
+import pytest
+import numpy as np
+
+
+@pytest.mark.skip(reason="Not a  test")
```

Comment:
There is no need to skip this function, [only modules/functions that have a name starting with `test_` are considered tests](https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery).